config/gulp/javascript.js

```diff
@@ -9,38 +9,30 @@ const rename = require("gulp-rename");
 const source = require("vinyl-source-stream");
 const sourcemaps = require("gulp-sourcemaps");
 const uglify = require("gulp-uglify");
+const babel = require("gulp-babel");
```

Comment:
> Some concerns about including `gulp-babel` since we're trying to get away from dependencies that don't update regularly.
> 
> Looking at their github, the last release was a year ago.

`gulp-babel` doesn't really do a whole lot, since it delegates most of the work to the underlying `@babel/core` peer dependency. The way I see it, a main risk for it becoming outdated would be if it keeps up with a future 8.0.0 release ([since it's pinned to `^7.0.0`](https://github.com/babel/gulp-babel/blob/46db7eefad83aee00de102128ba2745b687d3136/package.json#L47)).

Since gulp works with plain Node streams, it could be substituted with a stream. Babel doesn't come with a stream-based API, but we could emulate one with a little wrapper:

```diff
diff --git a/config/gulp/javascript.js b/config/gulp/javascript.js
index b549fc027..8edb6ce99 100644
--- a/config/gulp/javascript.js
+++ b/config/gulp/javascript.js
@@ -1,2 +1,3 @@
 /* eslint-disable arrow-body-style */
+const { Transform } = require("stream");
 const buffer = require("vinyl-buffer");
@@ -11,3 +12,3 @@ const sourcemaps = require("gulp-sourcemaps");
 const uglify = require("gulp-uglify");
-const babel = require("gulp-babel");
+const babel = require("@babel/core");
 const merge = require("merge-stream");
@@ -32,3 +33,17 @@ gulp.task(task, () => {
     stream
-      .pipe(babel({ presets: ["@babel/preset-env"] }))
+      .pipe(
+        new Transform({
+          objectMode: true,
+          async transform(file, _encoding, callback) {
+            const { code } = await babel.transformAsync(
+              file.contents.toString(),
+              { presets: ["@babel/preset-env"] }
+            );
+            // eslint-disable-next-line no-param-reassign
+            file.contents = Buffer.from(code);
+            this.push(file);
+            callback();
+          },
+        })
+      )
       .pipe(rename({ basename }))
```

This is basically a reimplementation of what `gulp-babel` does anyways.

Another option: We could write the `uswds-init.js` in ES5-safe JavaScript, then skip Babel for that file altogether, then revert back to `babelify` for the remaining entrypoint.